src/cli-task.js

```diff
@@ -3,11 +3,12 @@ const Stopwatch = require('./stopwatch');
 const runSeries = require('./run-series');
 const loadConfig = require('./load-config');
 const loadPackages = require('./load-packages');
+const { applyFilter } = require('./filter');
 
 module.exports = (task) => {
 	const timer = new Stopwatch();
 
-	return async (...args) => {
+	return async (cmd, options) => {
```

Comment:
I think the first arg here should be ignored and there is a third arg of `command`.

With command you can get the parsed options, e.g. `command.opts()` or `command.parent.opts()` to get global options.